tests/dat/actions/unicode.tests/swift-4.2.txt

```diff
@@ -15,8 +15,9 @@
  * limitations under the License.
  */
 
-func main(args: [String:Any]) -> [String:Any] {
-    if let str = args["delimiter"] as? String {
+func main(args: Any) -> Any {
+    let dict = args as! [String:Any]
+    if let str = dict["delimiter"] as? String {
```

Comment:
Found relative test cases failed in our  downstream, so need to fix it in upstream firstly.